keyboards/narfkb/info.json

```diff
@@ -0,0 +1,98 @@
+{
+    "manufacturer": "sepp",
+    "url": "https://github.com/sebastian-stumpf/narfkb",
+    "maintainer": "sebastian-stumpf",
+    "keyboard_name": "narfkb",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x1234",
+        "device_version": "1.0.0"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Comment:
```suggestion
        "nkro": true,
        "encoder": true
```

Reply:
would the `#ifdef ENCODER_ENABLE` in narfkb.c still function if this change was implemented?

Reply:
would the `#ifdef ENCODER_ENABLE` in narfkb.c still function if this change was implemented?